solarwinds_apm/configurator.py

```diff
@@ -808,10 +808,6 @@ def _create_init_event(
         keys: dict = None,
     ) -> Any:
         """Create a Reporter init event if the reporter is ready."""
-        if apm_config.is_lambda:
-            logger.debug("Skipping init event in lambda")
-            return None
-
```

Comment:
Removed this because `_configure` is the only caller of this `_create_init_event`, and the former checks if legacy already